ql/src/test/resources/hsmm/hsmm_cfg_01.yaml

```diff
@@ -0,0 +1,9 @@
+databaseIncludeLists:
```

Comment:
It took me some time to understand that `databaseIncludeLists` controls the tables too.
Maybe `migrationLists`? or something like this?